tests/e2e/specs/onboarding/step-3-complete-campaign.test.js

```diff
@@ -48,7 +48,35 @@ let page = null;
 
 test.describe( 'Complete your campaign', () => {
 	test.beforeAll( async ( { browser } ) => {
-		page = await browser.newPage();
+		// Use a dedicated context so the init script runs before any document scripts
```

Comment:
@ankitrox This test should not be updated at all since the changes should not affect tests for the usual onboarding flow. If we need to update those tests for them to pass, then there is something we are missing somewhere. There should not be any changes to this test file and it should continue pass.

Reply:
We need to update this slightly as we discussed because we need to configure the `glaData` values as per test scenario.